indico/modules/events/editing/client/js/components/ExtensionList.jsx

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+export default function ExtensionList({value, disabled, onChange, onFocus, onBlur}) {
+  const handleChange = (e, {value: newValue}) => {
+    onChange(_.uniq(newValue.map(x => x.trim().replace(/^\./, ''))));
```

Comment:
What if you have `....pdf` (which will break the backend validation)? Shouldn't it be `/^\.+/`?

Reply:
That's a good point, I fixed that. I've actually added `/^\**\.+/` on front end in case someone would write `*.txt`